src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj

```diff
@@ -27,6 +27,10 @@
     </ItemGroup>
   </Target>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(MicrosoftExtensionsOptionsConfigurationExtensionsVersion)" />
```

Comment:
>Microsoft.Extensions.Options.ConfigurationExtensions" [](start = 31, length = 53)

Looks 3.1.9 is the latest one align with dotnetcore versions. Please check if netstandard2.0 can be applied. Otherwise, it may need to split to different frameworks like line34~41.